byway/src/main/java/com/google/sps/servlets/UserLocations.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+import java.util.ArrayList;
+/*
+* Stores users start location and destinaions as they add additional places
+*/
+
+public class UserLocations{ 
+  private String start;
+  private ArrayList<String> destinations;
+  
+  public UserLocations(String start, ArrayList destinations) {
+    this.start = start;
+    this.destinations = destinations;
+  }
```

Comment:
Since you create `UserLocations` from a datastore `Entity` in multiple places, consider creating a method like 

`public static UserLocations fromEntity(Entity entity)`

So that that conversion exists only in one place.

Or add TODO for this so we don't have to do it in this PR (you can create a more narrow PR just for the single change later):

`// TODO(renau06): add "fromEntity" method to create UserLocations from a datastore entity.`

Reply:
added this  TODO for now